discussion_notes.md

```diff
@@ -0,0 +1,104 @@
+## Discussion
+
+**Why do all of these models seem to get worse quality predictions with larger context ratios?**
+- Longer context lengths can sometimes amplify noise or irrelevant patterns in the data. When the model tries to make predictions based on this noisy context, it may produce less reliable forecasts.
```

Comment:
Note for AK: The first four bullets here are different ways of looking at the same problem.